hexrdgui/main_window.py

```diff
@@ -604,10 +606,14 @@ def images_loaded(self, enabled=True):
     def on_action_open_materials_triggered(self):
         selected_file, selected_filter = QFileDialog.getOpenFileName(
             self.ui, 'Load Materials File', HexrdConfig().working_dir,
-            'HDF5 files (*.h5 *.hdf5)')
+            'HDF5 files (*.h5 *.hdf5), CIF (*.cif)')
+        if not selected_file:
```

Comment:
@psavery Currently only one CIF file can be opened at a time through the file menu since we only allow one HDF5 file to be selected at a time. Multiple CIF files can be imported at once from the material list editor. Let me know if we should adjust this behavior.